custom_types.go

```diff
@@ -1,13 +0,0 @@
-package owl
-
-import "encoding/json"
-
-type FlexibleInteger int
-
-func (fi *FlexibleInteger) UnmarshalJSON(b []byte) error {
```

Comment:
The place where this was used was removed, it'll need to be kept for the player number since they send back an empty string if the player isn't assigned a jersey number.

Reply:
Fair enough! If you create a branch I'll point this PR toward it and you can make these changes before merging into master. 👍

Reply:
I opened a develop branch, feel free to point it to that and I'll take a look at it.